libs/sdk-ui-dashboard/src/presentation/drill/DrillConfigPanel/DrillToUrl/CustomUrlEditorParametersSections/IdentifierParametersSection.tsx

```diff
@@ -0,0 +1,121 @@
+import { InsightIdParameterDetail } from "../ParameterDetails/InsightIdParameterDetail";
+import { ClientIdParameterDetail } from "../ParameterDetails/ClientIdParameterDetail";
+import { DataProductIdParameterDetail } from "../ParameterDetails/DataProductIdParameterDetail";
+import { Parameter } from "./Parameter";
+import { IIdentifierParametersSectionProps } from "../types";
+import { DRILL_TO_URL_PLACEHOLDER } from "../../../../../model";
+
+interface IIdentifierParameter {
+    titleIntlKey: string;
+    placeholder: string;
+}
+
+const identifierParametersSection: IIdentifierParameter[] = [
+    {
+        titleIntlKey: "configurationPanel.drillIntoUrl.editor.insightIdParameterLabel",
+        placeholder: DRILL_TO_URL_PLACEHOLDER.INSIGHT_ID,
+    },
+    {
+        titleIntlKey: "configurationPanel.drillIntoUrl.editor.widgetIdParameterLabel",
+        placeholder: DRILL_TO_URL_PLACEHOLDER.WIDGET_ID,
+    },
+    {
+        titleIntlKey: "configurationPanel.drillIntoUrl.editor.dashboardIdParameterLabel",
+        placeholder: DRILL_TO_URL_PLACEHOLDER.DASHBOARD_ID,
+    },
+    {
+        titleIntlKey: "configurationPanel.drillIntoUrl.editor.projectIdParameterLabel",
+        placeholder: DRILL_TO_URL_PLACEHOLDER.PROJECT_ID,
+    },
+    {
+        titleIntlKey: "configurationPanel.drillIntoUrl.editor.workspaceIdParameterLabel",
+        placeholder: DRILL_TO_URL_PLACEHOLDER.WORKSPACE_ID,
+    },
+    {
+        titleIntlKey: "configurationPanel.drillIntoUrl.editor.clientIdParameterLabel",
+        placeholder: DRILL_TO_URL_PLACEHOLDER.CLIENT_ID,
+    },
+    {
+        titleIntlKey: "configurationPanel.drillIntoUrl.editor.dataProductIdParameterLabel",
+        placeholder: DRILL_TO_URL_PLACEHOLDER.DATA_PRODUCT_ID,
+    },
+];
+
+const getDetailContent = (type: string, title: string): JSX.Element => {
+    switch (type) {
+        case DRILL_TO_URL_PLACEHOLDER.PROJECT_ID:
+        case DRILL_TO_URL_PLACEHOLDER.WORKSPACE_ID:
+            return <ProjectIdParameterDetail title={title} />;
+        case DRILL_TO_URL_PLACEHOLDER.DASHBOARD_ID:
+            return <DashboardIdParameterDetail title={title} />;
+        case DRILL_TO_URL_PLACEHOLDER.WIDGET_ID:
+            return <WidgetIdParameterDetail title={title} />;
+        case DRILL_TO_URL_PLACEHOLDER.INSIGHT_ID:
+            return <InsightIdParameterDetail title={title} />;
+        case DRILL_TO_URL_PLACEHOLDER.CLIENT_ID:
+            return <ClientIdParameterDetail title={title} />;
+        case DRILL_TO_URL_PLACEHOLDER.DATA_PRODUCT_ID:
+            return <DataProductIdParameterDetail title={title} />;
+        default:
+            return <></>;
+    }
+};
+
+export const IdentifierParametersSection: React.FC<IIdentifierParametersSectionProps> = ({
+    enableClientIdParameter,
+    enableDataProductIdParameter,
+    enableWidgetIdParameter,
+    onAdd,
+    intl,
+}) => {
+    const enableRenamingProjectToWorkspace = false; // TODO: use translation with | maybe?
```

Comment:
You can't do that, later in this file there is a direct comparison of the ids, the | translation would not help here I am afraid.